src/main/java/com/iota/iri/service/API.java

```diff
@@ -841,7 +874,7 @@ private synchronized AbstractResponse findTransactionsStatement(final Map<String
             final Set<String> bundles = getParameterAsSet(request,"bundles",HASH_SIZE);
             for (final String bundle : bundles) {
                 bundlesTransactions.addAll(
-                        BundleViewModel.load(instance.tangle, HashFactory.BUNDLE.create(bundle))
+                    BundleViewModel.load(instance.tangle, HashFactory.BUNDLE.create(bundle))
```

Comment:
this indentation is the same for the whole method (see line below), and is not related to the PR -- please revert